pyproject.toml

```diff
@@ -18,6 +18,7 @@ psycopg2-binary = "2.9.3"
 databases = "0.6.0"
 docxtpl = "0.16.0"
 python-multipart = "0.0.5"
+xlrd = "1.2.0"
```

Comment:
[5/6] RUN poetry install:
#0 1.762 Skipping virtualenv creation, as specified in config file.
#0 5.172 Installing dependencies from lock file
#0 8.392 Warning: poetry.lock is not consistent with pyproject.toml. You may be getting improper dependencies. Run poetry lock [--no-update] to fix it.       
#0 8.770
#0 8.770 Because e-university-api depends on xlrd (1.2.0) which doesn't match any versions, version solving failed.

Reply:
Ok, but according to "Updating dependencies to their latest versions" section in poetry documentation we must use the 'update' command for the poetry.lock file to be up-to-date. This moment hasn't reflected yet in our readme.

Reply:
Should I change the poetry.lock file?

Reply:
Run 'poetry update' command to change .lock file

Reply:
ill try, thanks, let u know about results

Reply:
I had similar problem with this library, and `poetry update` fixed it✅

Reply:
youp. it helped